app/js/components/systemMetadata/systemMetadata.jade

```diff
@@ -1,11 +1,104 @@
+            .alert-danger.text-center.pad5(ng-if='!system.isCheckingIn')
+              span(translate) This system is not checking in.
+
+      // Basic system meta-data table
+      .col-sm-5
+        table.meta-table
+          tbody
+            tr(ng-repeat='facts in initialMetadata')
+              td
+                b(translate) {{facts.label}}
+              td(translate) {{facts.value}}
+
+#expandableDataTable.expandable-data-table(ng-if='!loading; hasMetadata()', ng-class="{expanded: expandedTabs || expandIndefinite}")
+  // Toggle button
+  .toggle(ng-class="{expandedIndefinite: expandIndefinite}")
+    i.fa.fa-plus#iconSpin(ng-click="getData('network'); toggleTabsIndefinite(); rotateIcon()")
+
+  // Nav tabs
+  ul.nav(ng-class="{expandedIndefinite: expandIndefinite}", role='tablist', ng-mouseenter="expandTabContent()",
+         ng-mouseleave="tabCollapseTimer()", ng-click="expandTabsIndefinite()")
+    li(role='presentation', ng-if="!disableTab('system')")
+      a#system-tab(href='#system', aria-controls='system', role='tab', data-toggle='tab', ng-click="getData('system')")
+        i.fa.fa-desktop &nbsp;
+        span(translate).tabname System
+    li(role='presentation', ng-if="!disableTab('network')")
+      a#network-tab(href='#network', aria-controls='network', role='tab', data-toggle='tab', ng-click="getData('network')")
+        i.fa.fa-sitemap &nbsp;
+        span(translate).tabname Network
+
+  // Tab panes
+  .tab-content(ng-class="{hidden: !expandTabs && !expandIndefinite}", ng-init="setDefaultTab()")
+    #system.tab-pane(role='system')
+      .row
+        .col-sm-4(ng-repeat="data in tableData.data")
+          card
+            card-header.gray
+              b {{ data.label }}
+            card-content
+              ul.list.list-simple
+                li(ng-repeat="value in data.values", translate)
+                  strong {{value.label}}
+                  br
+                  span {{value.value}}
+
+
+
+          //table.singleColumnTable
+          //  thead
+          //    tr
+          //      th.ellipsis-overflow(ng-if=" === 'BIOS'", translate) BIOS
+          //      th.ellipsis-overflow(ng-if="data.label === 'System'", translate) System
+          //  tbody
+          //    tr
+          //      td(ng-repeat="value in data.values")
+          //        .dataName
+          //          span {{value.label}}
+          //        .dataValue
+          //          span {{value.value}}
+
+
+        //table.singleColumnTable
+        //  thead
+        //    tr
+        //      th.ellipsis-overflow(ng-if="data.label === 'BIOS'", translate) BIOS
+        //      th.ellipsis-overflow(ng-if="data.label === 'System'", translate) System
+        //  tbody
+        //    tr
+        //      td(ng-repeat="value in data.values")
+        //        .dataName
+        //          span {{value.label}}
+        //        .dataValue
+        //          span {{value.value}}
+    #network.tab-pane(role='network')
```

Comment:
I couldn't see a working example of this, but what I saw looked a bit odd with the scrollbar outside of the table and the css looks overly complicated. I can see potential issues with choosing this solution.

A simpler and cleaner way to accomplish the same functionality, is to return a table of 3 rows with the option to show the rest if they exist. Upon click > the remaining rows are loaded